homeassistant/components/ozw/websocket_api.py

```diff
@@ -1,21 +1,22 @@
 """Web socket API for OpenZWave."""
 
-import logging
-
 from openzwavemqtt.const import EVENT_NODE_ADDED, EVENT_NODE_CHANGED
 import voluptuous as vol
 
 from homeassistant.components import websocket_api
 from homeassistant.core import callback
+from homeassistant.exceptions import HomeAssistantError
 
-from .const import DOMAIN, MANAGER, OPTIONS
-
-_LOGGER = logging.getLogger(__name__)
+from .const import ATTR_CONFIG_PARAMETER, ATTR_CONFIG_VALUE, DOMAIN, MANAGER, OPTIONS
+from .entity import get_config_parameters, set_config_parameter
```

Comment:
I added them to entity because it's applicable to all entities, and it didn't really make sense anywhere else. We could add a `helpers.py` and move them there if that would make more sense from an organizational perspective

Reply:
In my view they are applicable to nodes. The only relation to entities is that all entities belong to some node. There is nothing in the code or API that relates to an entity. `helper.py` or `helpers.py` seem to be used by other components, so that would work. That said, since I'm the only one complaining, feel free to just leave it as is!

Reply:
A new module `node.py`?